### PR TITLE
HP-774: Upgraded lodash package and types

### PR DESCRIPTION
**Pull request by NikoHelle**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8980,11 +8985,16 @@ lodash.xorby@^4.7.0:
...
+"lodash@>=3.5 <5", lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.5, lodash@~4.17.10:
   version "4.17.20"
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.20.tgz#b44a9b6297bcb698f1c51a3545a2b3b368d59c52"
   integrity sha512-PlhdFcillOINfeV7Ni6oF1TAEayyZBoZ8bcshTHqOYJYlrqzRK5hagpagky5o4HfCzzd1TRkXPMFq6cKk9rGmA==
 
+lodash@^4.17.21:
+  version "4.17.21"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.21.tgz#679591c564c3bffaae8454cf0b3df370c3d6911c"
+  integrity sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg==
+
```
Comment (akikoskinen) [inline yarn.lock]: The version 4.17.21 is added in addition to version 4.17.20 so there are now two versions of `lodash`. But to me it seems that version 4.17.21 satisfies all the version requirements (lodash@>=3.5 <5", lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.5, lodash@~4.17.10, lodash@^4.17.21). So the version 4.17.20 isn't needed anymore? Can it be removed from `yarn.lock`?

File: yarn.lock [hunk 2]
```diff
@@ -2303,6 +2303,11 @@
   resolved "https://registry.yarnpkg.com/@types/lodash/-/lodash-4.14.165.tgz#74d55d947452e2de0742bad65270433b63a8c30f"
   integrity sha512-tjSSOTHhI5mCHTy/OOXYIhi2Wt1qcbHmuXD1Ha7q70CgI/I71afO4XtLb/cVexki1oVYchpul/TOuu3Arcdxrg==
 
+"@types/lodash@^4.14.170":
+  version "4.14.170"
+  resolved "https://registry.yarnpkg.com/@types/lodash/-/lodash-4.14.170.tgz#0d67711d4bf7f4ca5147e9091b847479b87925d6"
+  integrity sha512-bpcvu/MKHHeYX+qeEN8GE7DIravODWdACVA1ctevD8CN24RhPZIKMn9ntfAsrvLfSX3cR5RrBKAbYm9bGs0A+Q==
```
Comment (akikoskinen) [inline yarn.lock]: Version 4.14.170 satisfies all the version requirements so 4.14.165 doesn't have to be in the `yarn.lock` file any more?

**Review by akikoskinen — CHANGES_REQUESTED** [review]
To me it seems that one version of each package should be enough.

File: yarn.lock (hunk 1 above)
Comment (NikoHelle) [reply yarn.lock]: Yarn lock should not be edited manually. This seems to be an issue with yarn. I re-installed packages by removing lock file and node_modules and this was fixed. But it changed a lot of other dependencies too that are not related to this PR. 

So I did the same with "develop"-branch and created another PR: https://github.com/City-of-Helsinki/open-city-profile-ui/pull/179. I'll rebase this branch to that and run "yarn install" again. That will probably fix this.

Comment (NikoHelle) [reply yarn.lock]: Ran yarn with "clean" install and looks better now

File: yarn.lock (hunk 2 above)
Comment (NikoHelle) [reply yarn.lock]: Ran yarn with "clean" install and looks better now. This file should not be edited manually.

File: yarn.lock (hunk 1 above)
Comment (akikoskinen) [reply yarn.lock]: > Yarn lock should not be edited manually.

Yep, I meant that "force yarn to do the correct thing somehow". That's what you did. Well done :+1:

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**